Sources/SwifterSwift/Dispatch/DispatchQueueExtensions.swift

```diff
@@ -41,6 +41,19 @@ public extension DispatchQueue {
 
         return DispatchQueue.getSpecific(key: key) != nil
     }
+    
+    /// SwifterSwift: Runs passed closure asynchronous after certain time interval
+    ///
+    /// - Parameter delay: The time inverval after which the closure will run.
+    /// - Parameter qos: Quality of service at which the work item should be executed.
+    /// - Parameter flags: Flags that control the execution environment of the work item.
+    /// - Parameter work: The closure to run after certain time interval.
```

Comment:
There's a nice option in Xcode for creating documentation easily: Editor -> Structure -> Add Documentation
```suggestion
    /// SwifterSwift: Runs passed closure asynchronous after certain time interval
    /// - Parameters:
    ///   - delay: The time inverval after which the closure will run.
    ///   - qos: Quality of service at which the work item should be executed.
    ///   - flags: Flags that control the execution environment of the work item.
    ///   - work: The closure to run after certain time interval.
```